assets/app/views/project.html

```diff
@@ -102,9 +102,7 @@ <h2 ng-if="displayRouteByService[serviceId]" ng-init="otherRoutes = (routesBySer
                   </span>
                   <span class="small" ng-if="otherRoutes">
                     (and
-                    <a href="project/{{projectName}}/browse/services#service-{{serviceId}}">
-                      {{otherRoutes}} other {{(otherRoutes == 1) ? 'route' : 'routes'}}
-                    </a>)
+                    <a href="project/{{projectName}}/browse/services#service-{{serviceId}}">{{otherRoutes}} other {{(otherRoutes == 1) ? 'route' : 'routes'}}</a>)
```

Comment:
i know @benjaminapetersen fixed this URL in his branch so i'm guessing there will be a merge conflict here one way or the other